tests/acceptance/test_dcos_zookeeper.py

```diff
@@ -24,3 +24,8 @@ def test_delete_zk_node():
     uninstall_package_and_wait('marathon')
 
     assert delete_zk_node('universe/marathon-user')
+
+
+def get_zk_node_data():
+    install_package_and_wait('marathon')
+    get_zk_node_data('universe/marathon-user')
```

Comment:
one of the challenges for a shakedown user in using the python lib is understanding what the json objects are coming back.   With many of the mesos / marathon json objects there is often some level of documentation or help.   In this case a json object is expected back.   That's all an end user knows.   The reference link provided doesn't provide any help either.   It is useful IMO in the absence of documentation to have a test that provides some level of assertion of what can be expected.   At this point, it is require for a first time user to either hunt for an exhibitor json example in the wild or to run something similar to a copy of the test and print the output.